.eslintrc.json

```diff
@@ -38,16 +38,20 @@
     "import/no-extraneous-dependencies": "error",
     "import/order": "warn",
     "react/button-has-type": "error",
-    "@typescript-eslint/explicit-module-boundary-types": "off"
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@next/next/no-img-element": "off"
```

Comment:
I'm disabling this rule after wrestling with the `next/image` component and because it is [not supported in static exports](https://nextjs.org/docs/messages/export-image-api) without implementing a custom loader. We can still _use_ `next/image`, I just don't want ESLint telling us we should _always_ use it when it doesn't work for our current use case. my thought is let's come back to this once we know more about our deployment / CDN situation (there is an [akamai loader](https://nextjs.org/docs/basic-features/image-optimization#loader) we might be able to use)